hibernate-core/src/main/java/org/hibernate/engine/spi/ActionQueue.java

```diff
@@ -1268,7 +1268,9 @@ private void addParentChildEntityNames(AbstractEntityInsertAction action, BatchI
 				for ( int i = 0; i < propertyValues.length; i++ ) {
 					Object value = propertyValues[i];
 					Type type = propertyTypes[i];
-					addParentChildEntityNameByPropertyAndValue( action, batchIdentifier, type, value );
+					if ( value != null ) {
+						addParentChildEntityNameByPropertyAndValue( action, batchIdentifier, type, value );
+					}
```

Comment:
I suppose this change should be replicated in Hibernate Reactive's `ReactiveActionQueue`.

Is it just this one line, @NathanQingyangXu ?

Reply:
it is

Reply:
alright I will just copy it over then

Reply:
Ah, I failed to notice we should also remove the unnecessary nullness checking in `addParentChildEntityNameByPropertyAndValue` private method. It would hurt the code readability. IMHO, the ActionQueue has big room for improvement. I am gonna create another ticket and PR to include the above code change mentioned, together with many other minor issues.

Reply:
I created a ticket here: https://hibernate.atlassian.net/browse/HHH-14248. Will work on it soon.

Reply:
It is my fault. @Sanne 's code change is correct. Removing the nullness checking in `addParentChildEntityNameByPropertyAndValue` breaks FumTest for there is other entrance to the method bring null value in.